react/week-20/lesson.md

```diff
@@ -424,11 +422,11 @@ From this we can get a clue about when to use state. If data *changes over time*
 
 ### Container components
 
-In real world applications, the things we want to remember in state follow the *business logic* required by our users. So for example the number of bookings in the exercise above increases when you add a booking. Most of the time, business logic is about figuring out when and how to change state.
+In real world applications, the things we want to remember in state follow the *business logic* required by our users. So for example the number of caught Pokemon in the exercise  increases when you click on the button *Catch Pokemon*. Most of the time, business logic is about figuring out when and how to change state.
```

Comment:
Oops, thanks for catching those - hang overs from the old examples that I updated.